examples/03_connectivity/plot_sphere_based_connectome.py

```diff
@@ -110,9 +114,11 @@
 # partial correlations.
 
 from nilearn.connectome import ConnectivityMeasure
-connectivity_measure = ConnectivityMeasure(kind='partial correlation')
-partial_correlation_matrix = connectivity_measure.fit_transform(
-    [time_series])[0]
+
+connectivity_measure = ConnectivityMeasure(kind="partial correlation")
+partial_correlation_matrix = connectivity_measure.fit_transform([time_series])[
+    0
```

Comment:
:-(

Reply:
yup we will inevitably get those weird line splits, but that would be the case with any line length.